### ISSUE TITLE
error

### ISSUE THREAD

**Issue by ASHISHKATARUKA** [microsoft/vscode-copilot-release#7723]
Type: <b>Bug</b>

Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}. Error Component Stack
    at DescriptionWarning (Dialog.tsx:531:66)
    at Dialog.tsx:383:13
    at Dialog.tsx:258:58
    at Presence (Presence.tsx:13:11)
    at Dialog.tsx:233:64
    at Slot.tsx:61:11
    at Slot.tsx:13:11
    at Primitive.tsx:38:13
    at Portal.tsx:22:22
    at Presence (Presence.tsx:13:11)
    at Provider (createContext.tsx:58:15)
    at DialogPortal (Dialog.tsx:145:11)
    at _c1 (dialog.tsx:33:6)
    at Provider (createContext.tsx:58:15)
    at Dialog (Dialog.tsx:52:5)
    at div (<anonymous>)
    at div (<anonymous>)
    at header (<anonymous>)
    at div (<anonymous>)
    at Home (home.tsx:42:39)
    at RenderedRoute (hooks.tsx:657:26)
    at Routes (components.tsx:504:3)
    at Suspense (<anonymous>)
    at PostgreSQLDatabaseProvider (PostgreSQLDatabaseContext.tsx:75:87)
    at App (<anonymous>)
    at Router (components.tsx:413:13)
    at BrowserRouter (index.tsx:774:3)
overrideMethod @ hook.js:608
hook.js:608  Warning: Missing `Description` or `aria-describedby={undefined}` for {DialogContent}.

Extension version: 0.27.2025041102
VS Code version: Code - Insiders 1.100.0-insider (043f699013796cbeae7c8fe9abac9b9eb26b3c51, 2025-04-11T05:23:04.452Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|13th Gen Intel(R) Core(TM) i7-1360P (16 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.73GB (0.70GB free)|
|Process Argv|--crash-reporter-id b08c764b-c4fc-4487-93d5-fbc0351dd4d7|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt551cf:31249598
vscod805cf:30301675
c4g48928:30535728
962ge761:30841072
dsvsc014:30777825
dsvsc015:30821418
h48ei257:31000450
pythontbext0:30879054
cppperfnew:30980852
dwnewjupytercf:31046870
pythonrstrctxt:31093868
nativeloc1:31118317
e80f6927:31120813
dwcopilot:31158714
5b1c1929:31184661
6074i472:31201624
dwoutputs:31242946
customenabled:31248079
9064b325:31222308
copilot_t_ci:31222730
g012b348:31231168
cp15370_c:31280196
copilothoveron:31262361
pythoneinst12:31251391
treesitterini:31278752
bgtreat:31252123
c7cif404:31283753
31787653:31262186
3e8i5726:31271747
pythonpulldiagcf:31258011
6gi0g917:31259950
61ibe868:31282443
996jf627:31264550
pythonrdcb7:31268811
gh9da704:31280609
747dc170:31275146
aj496949:31278748
pylancecolorcf:31281600
generatesymbolt:31280541
convertfstringf:31280702
gendocf:31280703

```

</details>
